next.config.js

```diff
@@ -1,19 +1,9 @@
-const withSass = require('@zeit/next-sass')
-const withCss = require('@zeit/next-css')
-const withSourceMaps = require('@zeit/next-source-maps')()
+const path = require('path')
 
 module.exports = {
-  webpack5: false,
-  pageExtensions: ['page.js'],
```

Comment:
Avoid removing this — doing this is going to break pages that use the `index.page.js` suffix. e.g. The `/scan` route.

Reply:
I have reverted `pageExtensions: ['page.js']` but webpack 4 is not supported anymore.